packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -1578,49 +1648,64 @@ contract Deploy is Deployer {
         }
 
         uint32 rawGameType = GameType.unwrap(_params.gameType);
+        FaultDisputeGameParams memory _params_ = _params;
```

Comment:
Why is this required?

Reply:
ran into stack too deep errors

Reply:
Ah weird, can you leave a comment to that effect?